wasm/scripts/worker.js

```diff
@@ -30,26 +34,67 @@ const imports = {
     post_to_main: function(msg, len) {
         this.postMessage(arr_from_mem(msg, len));
     },
+    get_canvas_size: function() {
+        return (canvas.width << 16) | canvas.height;
+    },
+    set_canvas_size: function(w, h) {
+        canvas.width = w;
+        canvas.height = h;
+    },
+    gl_get_error: function() {
+        return gl.getError();
+    },
+    gl_create_vertex_array_and_buffer_with_data: function(ptr, len) {
+        // ATTENTION: The pointer must be 32-bit aligned.
+        const vao = gl.createVertexArray();
+        if (!(vao instanceof WebGLVertexArrayObject)) { return 1; }
```

Comment:
I don't know how we would share these numbers. I think this is fine. We could document this at the rust bindings module, so that this is also documented by `cargo-doc`.

Reply:
There are solutions, but depending on how likely those values are to change and whether they have implications on other code it might be beneficial to abstain from adding furher complexity